.github/workflows/build-and-test.yml

```diff
@@ -62,10 +62,18 @@ jobs:
             --image=${{ steps.image-name.outputs.name }} \
             -Djib.console=plain
 
-      - name: Run Trivy vulnerability scanner
-        uses: broadinstitute/dsp-appsec-trivy-action@v1
-        with:
-          image: ${{ steps.image-name.outputs.name }}
+#  dispatch-trivy:
```

Comment:
Because the Trivy.yml doesn't exist yet, this step will fail. Once this PR is merged, a follow-up PR will uncomment these.